Code changes for SNAP-2989: Snappy UI rebranding to Tibco ComputeDB iff it's Enterprise Edition

snappy-sachin opened the pull request:
## What changes were proposed in this pull request?

1. SnappyData is Community Edition

 - Displays Pulse logo on top left side.
 - Displays SnappyData logo on top right side.
 - About Box : Displays version and copyright information

2. TIBCO ComputeDB is Enterprise :

 - Displays TIBCO ComputeDB logo on top left side.
 - About Box  : - Displays version and copyright information

## How was this patch tested?

 - Tested Manually

Please review http://spark.apache.org/contributing.html before opening a pull request.

snappy-sachin committed:
Code changes for SNAP-2989:

1. SnappyData is Community Edition

 - Displays Pulse logo on top left side.
 - Displays SnappyData logo on top right side.
 - About Box : Displays version and copyright information

2. TIBCO ComputeDB is Enterprise :

 - Displays TIBCO ComputeDB logo on top left side.
 - About Box  : - Displays version and copyright information

dshirish reviewed (approved):
LGTM. Please get the screenshots reviewed from all relevant folks.

snappy-sachin committed:
Code changes for SNAP-2989:
 - Adding TIBCO ComputeDB logo with 30% reduced size.
 - CSS is updated
 - Product name correction

snappy-sachin committed:
Changes for SNAP-2989:
 - Product Name changes as per latest discussions.
 - Updating product documentation links as per latest communications.
